src/components/AnonymousChat/child/ChannelTitle.tsx

```diff
@@ -45,7 +45,7 @@ const ChannelTitle = ({
       marginRight: 4
     },
     chatContentUnreadCountContainer: {
-      backgroundColor: isAnonymousTab ? colors.bondi_blue : colors.darkBlue,
+      backgroundColor: isAnonymousTab ? colors.anon_primary : colors.darkBlue,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `colors.bondi_blue` to `colors.anon_primary` seems fine as long as the `colors.anon_primary` is defined in your colors object and it provides the desired color for the unread count container in anonymous mode. If not, you might want to define it or use an existing color that fits your design requirements.
```diff
-      backgroundColor: isAnonymousTab ? colors.bondi_blue : colors.darkBlue,
+      backgroundColor: isAnonymousTab ? colors.anon_primary : colors.darkBlue,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->